Libplanet/Net/Swarm.cs

```diff
@@ -1330,12 +1374,20 @@ CancellationToken cancellationToken
             try
             {
                 _logger.Debug($"Trying to DialAsync({peer.EndPoint})...");
-                await DialAsync(
+                Pong pong = await DialAsync(
                     ToNetMQAddress(peer),
                     dealer,
                     cancellationToken);
                 _logger.Debug($"DialAsync({peer.EndPoint}) is complete.");
 
+                if (pong.ProtocolVersion != _protocolVersion)
+                {
+                    dealer.Dispose();
+                    throw new DifferentProtocolVersionException(
+                        $"Peer protocol version({pong.ProtocolVersion}) is " +
```

Comment:
it would be better if current version(`_protocolVersion`) exists in the error message.

Reply:
I added the current version in the error message. @longfin